integration/samples.sh

```diff
@@ -27,7 +27,7 @@ do
         node dist/cli/main.js -p ${P_JSON}
     elif [ -f $P ]; then
         echo "Building sample ${P}..."
-        if [ $P='failures' ]; then
+        if [[ $P = *'fail-'* ]]; then
```

Comment:
@alan-agius4, this line used to have an issue, the old code was never using the else branch because:
```
$ P="abcd"

$ [ $P="failures" ] && echo "TRUE" || echo "FALSE"
TRUE

$ [ $P = "failures" ] && echo "TRUE" || echo "FALSE"
FALSE

$ [ $P = "abcd" ] && echo "TRUE" || echo "FALSE"
TRUE

$ [ $P = "bc" ] && echo "TRUE" || echo "FALSE"
FALSE

$ [[ $P == *"bc"* ]] && echo "TRUE" || echo "FALSE"
TRUE
```
The new code is bash specific, but since the script bang is `#!/bin/bash`, I assume that we can go with it. If we want to support more shells, we should change the bang to `#!/bin/sh` and use a POSIX compliant comparison here.

Also, the `$FIND_COMMAND` from the else branch is wrong since it uses glob expansion and not executing the bash command. And the `win32` branch might also have an issue, it doesn't look just into the samples directory. I'll take a look at this script in the future and I'll create a PR.

Reply:
This change no longer exists, it has been fixed in #1846